.github/workflows/stress.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           { config: "Debug", plat: "macos", os: "macos-12", arch: "x64", tls: "openssl3", build: "-Test" },
           { config: "Debug", plat: "linux", os: "ubuntu-20.04", arch: "x64", tls: "openssl", sanitize: "-Sanitize", build: "-Test" },
           { config: "Debug", plat: "linux", os: "ubuntu-20.04", arch: "x64", tls: "openssl3", sanitize: "-Sanitize", build: "-Test" },
+          { config: "Debug", plat: "linux", os: "ubuntu-22.04", arch: "x64", tls: "openssl3", build: "-Test" },
```

Comment:
XDP doesn't work with sanitizer? Why?

Reply:
I previously saw data alignment related error.

Reply:
We should follow up (in a later PR) on the sanitizer.